vue-components/stories/BouncingDots.stories.ts

```diff
@@ -0,0 +1,39 @@
+import BouncingDots from '@/components/BouncingDots';
+import { bouncingDotsSizes } from '@/components/bouncingDots';
+import { Component } from 'vue';
+
+export default {
+    component: BouncingDots,
+    title: '/Loader/BouncingDots',
+};
+
+export function basic( args: { content: string; underlined: boolean } ): Component {
```

Comment:
I believe this page can be called "All", since it contains all size variants

Reply:
And since this is supposed to not contain controls, we con drop the `args` parameter and every place where it is used.

Reply:
removed unnecessary `args` parameter